src/Adapter/Product/Repository/ProductPackRepository.php

```diff
@@ -30,7 +30,7 @@
 
 use Pack;
 use PrestaShop\PrestaShop\Core\Domain\Product\Combination\ValueObject\CombinationId;
-use PrestaShop\PrestaShop\Core\Domain\Product\Exception\ProductPackException;
+use PrestaShop\PrestaShop\Core\Domain\Product\Pack\Exception\ProductPackException;
```

Comment:
As you introduced new dir for Pack, we should move in commands here too.
Could you add a reminder somewhere? (or move them in this pr)

Reply:
It might even worth to move those namespace and exception introduction changes to separate PR, because this one is already HEAVY (btw im in need of those Pack changes in another PR already :smile: )

Reply:
I added the todo for the commands to avoid making this one even bigger However since the exceptions are already moved and behat test already rely on them I prefer to keep them as is in this PR